webapp/sitemap.py

```diff
@@ -267,65 +261,56 @@ def build_navigation(self, current_path=None):
         if os.path.exists(config_path):
-            with open(config_path, 'r') as navigation_file:
+            with open(config_path, "r") as navigation_file:
                 config = yaml.load(navigation_file)
 
         sitemap = self.get()
 
         options = DEFAULT_NAVIGATION_OPTIONS
-        if isinstance(config, dict) and '_items' in config:
-            config_items = config['_items']
+        if isinstance(config, dict) and "_items" in config:
+            config_items = config["_items"]
 
             # TODO: Abstract duplicate logic from _populate_navigation
-            if '_options' in config:
-                options = config['_options']
+            if "_options" in config:
+                options = config["_options"]
             # Set options for only root node
-            if '_options_local' in config:
-                local_options = config['_options_local']
-                if 'nesting_limit' in local_options:
-                    remaining_node_depth = local_options['nesting_limit']
-                    options['remaining_depth'] = remaining_node_depth
+            if "_options_local" in config:
+                local_options = config["_options_local"]
+                if "nesting_limit" in local_options:
+                    remaining_node_depth = local_options["nesting_limit"]
+                    options["remaining_depth"] = remaining_node_depth
         else:
             config_items = config
 
         # Determine current path and root
-        current_path = current_path.strip('/')
+        current_path = current_path.strip("/")
         root_path_key = None
         if current_path:
-            root_path_key = current_path.split('/')[0]
-        options['root_path'] = '/{path}'.format(
-            path=root_path_key or ''
-        )
+            root_path_key = current_path.split("/")[0]
+        options["root_path"] = "/{path}".format(path=root_path_key or "")
 
         sorted_tree = self._populate_navigation(
-            config_items,
-            sitemap,
-            options=options
+            config_items, sitemap, options=options
         )
 
         if current_path:
             self._set_active_navigation_items(sorted_tree, current_path)
         if root_path_key:
             root_node = self._find_navigation_item(sorted_tree, root_path_key)
             if root_node:
-                if root_node.get('children'):
+                if root_node.get("children"):
                     sorted_tree = [root_node]
-                    back_link = [{
-                        'type': 'back',
-                        'path': '/',
-                    }]
+                    back_link = [{"type": "back", "path": "/"}]
                     sorted_tree = back_link + sorted_tree
-                elif root_node.get('type') == 'heading':
-                    print('2')
-                    root_node['type'] = None
+                elif root_node.get("type") == "heading":
+                    print("2")
```

Comment:
not from this PR but you can remove this line

Reply:
I'm impressed - very thorough of you to find that.

However, in future, if you have a drive-by fix you want to suggest someone do, could you consider approving the PR anyway, with just adding the drive-by as a suggestion? You could even file your own PR to remove that line, or file an issue to make sure it doesn't get forgotten - either would avoid delaying the first PR. We shouldn't be forcing people do work beyond the scope of the PR if we can possibly avoid it.

E.g.: I was hoping to simply merge this now so that Renovate could have rebased all its other PRs by the time I got into the office, but it's not easy to get to a terminal right now to add this fix.

Reply:
Yeah I was checking if it was still working and I had this printed in my logs, that's how I got it.

But no worries I approve your PR and let's remove it in another small one.

Reply:
Thanks :)